base/libgit2/libgit2.jl

```diff
@@ -132,11 +149,24 @@ function diff_files(repo::GitRepo, branch1::AbstractString, branch2::AbstractStr
     return files
 end
 
+"""
+    is_ancestor_of(a::AbstractString, b::AbstractString, repo::GitRepo) -> Bool
+
+Returns `true` if `a`, a [`GitHash`](@ref) in string form, is an ancestor of
+`b`, a [`GitHash`](@ref) in string form. `a` and `b` could be branches, tags,
```

Comment:
You do not refer to it by name, no matter if it's a commit/branch/tag. `a` can be a commit and `b` can be a tag, even.